includes/Core/Assets/Assets.php

```diff
@@ -894,22 +894,49 @@ private function get_external_assets() {
 			// 'fallback'     => true,
 			// )
 			// ),
-			// new Script(
-			// 'wp-api-fetch',
-			// array(
-			// 'src'          => $base_url . 'js/externals/apiFetch.js',
-			// 'dependencies' => array( 'wp-i18n', 'wp-polyfill', 'wp-url' ),
-			// 'version'      => '3.6.4',
-			// 'fallback'     => true,
+			new Script('e2e-utilities', array(
```

Comment:
This shouldn't be registered in the main plugin. It should go in an mu-plugin which is only used in e2e tests. See https://github.com/google/site-kit-wp/tree/develop/tests/e2e/mu-plugins

Reply:
Indeed! I'm in a bit of refactoring hell at the moment so this stuff isn't final, but I'll move it soon just for tidiness. 😄